src/midi.cpp

```diff
@@ -86,12 +88,17 @@ static void HandleNoteOff(byte channel, byte note) { HandleNoteOn(channel, note,
 
 static void HandleControlChange(byte channel, byte data1, byte data2) {
 	leftDot();
-	if (channel == masterChannel) {
+
+	if((channel ==16)&&(data1==85)){EEPROM.update(3994,data2);// settings (set by online webMIDI tool)
+	settings=data2;
+	}
+
+	else if (channel == masterChannel) {
 		if (data1 == 59)
 			data1 = 32;
 
 		if (1 <= data1 && data1 <= 36) {
-			int mapping[] = {-1, 12, 4,  6,  14, 1,  5, 15, 13, 16, 24, 26, 17, 27, 22, 25, 23, 20, 30,
+			int mapping[] = {-1, 33, 4,  6,  14, 1,  5, 15, 13, 16, 24, 26, 17, 27, 22, 25, 23, 20, 30,
```

Comment:
If going for the "no `case 33` in `ui_pots.cpp` approach, change the `33` to `-1` here.

Reply:
sorry I made the changes before I scrolled to your suggestions - will read all the review in future!